lib/pylightio/lookingglass/devices.py

```diff
@@ -282,6 +282,87 @@ class LookingGlassPortrait(LookingGlassDeviceMixin, BaseDeviceType):
     # NOTE: Here is the place to define methods required by the BaseClass for any
     #       device type implementations
 
+    # This type uses the LookingGlassBaseType and has no special requirements
+# 4K Gen2: 4K Gen2 Looking Glass
+class LookingGlass4kGen2(LookingGlassDeviceMixin, BaseDeviceType):
+
+    # PUBLIC MEMBERS
+    # ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+    type = "4k_gen2"                      # the unique identifier string of this device type
+    name = "4k Gen2 Looking Glass"        # name of this device type
+    formats = [LookingGlassQuilt]    # list of lightfield image formats that are supported
+    emulated_configuration = {       # configuration used for emulated devices of this type
+                'buttons': [0, 0, 0, 0],
+                'calibration': {
+                                    'configVersion': '1.0',
+                                    'serial': 'LKG-ADUMMY',
+                                    'DPI': 283.0,
+                                    'screenH': 4320.0,
+                                    'screenW': 7680.0,
+                                    'viewCone': 40.0,
+                                    'aspect': 1.77777777,
```

Comment:
Just to be sure: Is it correct that the 4k and 8k devices have the same screenH, screenW, and aspect ratio?

Reply:
Quite right! Fixed in the last commit.